README.md

```diff
@@ -46,7 +53,10 @@ var Panda = function(name, age) {
 ```
 Your Answer:
 ```js
```

Comment:
Close! Just remember we want to add `num_bamboo_eaten` by **1** so it would be 
```
Panda.prototype.eat_bamboo = function(){
  this.num_bamboo_eaten += 1;
}
```